components/common/Main.tsx

```diff
@@ -26,7 +37,7 @@ const Main = ({ children }: MainProps): JSX.Element => {
                 justifyContent="center"
             >
                 <LeftNav
-                    user={user}
+                    userWithRole={{ user, role }}
```

Comment:
Just a side note: Should we actually store `role` in `FirebaseAuthContext` instead of a locally scoped `const`?

Reply:
It possible to refactor to be in context in future.